### ISSUE TITLE
searching by author name

### ISSUE THREAD

**Issue by matchaginseng** [writingatlas/feedback#3]
Would be convenient to search / filter by author name under the AUTHORS tab, instead of just sorting!

**Comment by jenny8lee** [comment]
Agreed. We have a search toolbar in the side bar, but we should probably
add that in the authors page. Noted.
